package.json

```
@@ -7,14 +7,14 @@
     "build-css": "sass src:static/css --load-path=node_modules --style=compressed && postcss --use autoprefixer --replace 'static/css/**/*.css' --no-map",
     "watch-css": "yarn build-css && sass --load-path=node_modules -w src:static/css --style=compressed",
     "format-python": "black --line-length 79 webapp",
-    "test": "yarn run lint-python && yarn run test-python",
-    "test-python": "python3 -m unittest discover tests",
+    "test": "yarn run lint-python && pytest",
```

We could keep this as running `test-python` script, so there is only one place that defines the testing command

```suggestion
    "test": "yarn run lint-python && yarn run test-python",
```